server/bleep/src/webserver/studio.rs

```diff
@@ -389,3 +396,94 @@ async fn generate_llm_context(app: Application, context: Vec<ContextFile>) -> Re
 
     Ok(s)
 }
+
+#[derive(serde::Deserialize)]
+pub struct Import {
+    pub thread_id: Uuid,
+}
+
+/// Returns a new studio UUID.
+pub async fn import(
+    app: Extension<Application>,
+    user: Extension<User>,
+    Query(params): Query<Import>,
+) -> webserver::Result<String> {
```

Comment:
```suggestion
) -> webserver::Result<Uuid> {
```

can we return a uuid here? it implements `Serialize`.